xapian-core/cluster/cluster.cc

```diff
@@ -54,41 +60,46 @@ MSetDocumentSource::MSetDocumentSource(const MSet & mset_, doccount maxitems_)
 
 Document
 MSetDocumentSource::next_document() {
```

Comment:
The way I've thought of the convention while writing code was to put opening braces on the same line when its a function and on a different line when its a constructor. Have I misinterpreted things here?
Should I be necessarily adding this brace on a next line? Because that will require me to change other functions that I had written in a similar way.